examples/gallery/lines/linestrings.py

```diff
@@ -25,7 +25,7 @@
 
 # Convert object to EPSG 4326 coordinate system
 gdf = gdf.to_crs("EPSG:4326")
-print(gdf.head())
+gdf.head()
```

Comment:
Hm. My background off using `print` was that if running the complete script as "normal" Python script no output is shown, only when running this specific section separately (see comment https://github.com/GenericMappingTools/pygmt/pull/3711#discussion_r1899736562).

Reply:
Yes, output the data head information makes little sense when running the example as a normal Python script.